src/main/java/org/dependencytrack/model/Analysis.java

```diff
@@ -76,22 +76,22 @@ public class Analysis implements Serializable {
     @Persistent(defaultFetchGroup = "true")
     @Column(name = "STATE", jdbcType = "VARCHAR", allowsNull = "false")
     @NotNull
-    private AnalysisState analysisState;
+    private AnalysisState state;
```

Comment:
I understand the itch to rename these fields, but in doing so we're introducing a breaking API change (because fields in REST API responses change, too), and clients including the frontend will need to be updated. I'd prefer to avoid this if possible.

Reply:
Right, I'll update the fields in query instead 👍🏼